CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ project(maliit-framework VERSION 2.0.0)
 
 option(enable-docs "Build documentation" ON)
 option(enable-tests "Build tests" ON)
+option(enable-examples "Build examples" ON)
```

Comment:
I'd turn them off by default

Reply:
I agree. Else looks good, but OFF the default